shell/platform/windows/egl/egl.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_EGL_EGL_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_EGL_EGL_H_
+
+#include <string>
+
+namespace flutter {
+namespace egl {
+
+/// Log the last EGL error with an error message.
+void LogEGLError(std::string message);
```

Comment:
Consider `std::string_view` which is lighter-weight than a string copy.

Reply:
Ah right! Updated to use `std::string_view`